src/System.CommandLine.Hosting.Tests/System.CommandLine.Hosting.Tests.csproj

```diff
@@ -15,6 +15,9 @@
 
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="5.10.3" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)'!='netcoreapp3.1'">
     <PackageReference Include="Microsoft.Extensions.Configuration.CommandLine" Version="2.2.0" />
```

Comment:
Previously in .NET Standard 2.0 (and thus also the `net462` target), individual Generic Host packages needed to be referenced individually, since `Microsoft.Extensions.Hosting` only contained the bare minimum.

`Microsoft.Extensions.Configuration.CommandLine` is actually trasitively references by `Microsoft.Extensions.Hosting` in .NET Standard 2.1 (and .NET Core 3.0 and later)